services/ui-src/src/components/pages/Admin/AdminPage.tsx

```diff
@@ -41,23 +47,23 @@ export const AdminPage = () => {
     setIsBannerActive(bannerActivity);
   }, [bannerData]);
 
-  useEffect(() => {
-    setError(errorMessage);
-  }, [errorMessage]);
+  // useEffect(() => {
+  //   setError(errorMessage);
+  // }, [errorMessage]);
```

Comment:
Does this need to stay in?

Reply:
Removed as its no longer needed with Zustand :D